src/extension.ts

```diff
@@ -98,8 +97,7 @@ export async function activate(context: ExtensionContext) {
                         name: runTerminalName,
                         cwd: command[0]
                     });
-                    else if (lastWorkingDirectory !== command[0]) terminal.sendText('cd ' + command[0]);
-                    lastWorkingDirectory = command[0];
+                    terminal.sendText('cd ' + command[0]);
```

Comment:
I haven't tried this yet, but if `command[0]` is not an absolute path, then wouldn't the `cd` fail in case the shell is not in the right directory to begin with?

Reply:
Wouldn't a more obvious fix be to remove the if statement on line 97, always creating a new terminal regardless of whether one exists already?

Reply:
`command[0]` is an absolute path. Only the command itself uses a possibly relative path.

The problem with creating a new terminal each time is that they can accumulate and cause clutter, as shown below:

![Screenshot from 2022-04-26 22-06-47](https://user-images.githubusercontent.com/33707478/165444708-2a0cb657-d082-414b-b526-713534fd9ab2.png)

Reply:
Gotcha. Yes, if `command[0]` is absolute, then this will work. Great! Thanks for the fix!